components/ClientInfo/ClientInfo.tsx

```diff
@@ -0,0 +1,146 @@
+import styles from './ClientInfo.module.css';
+import { FiCheck, FiExternalLink } from "react-icons/fi";
```

Comment:
i'm not sure why this happens but some of the imports remain unresolved when i first pulled from the branch (react icons, @emotions/styled, etc.). i had to manually install them before getting the code to compile